config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml

```
@@ -22,6 +22,29 @@ periodics:
       args:
       - -c
       - "cd groups && make run -- --confirm"
+- name: ci-k8sio-vuln-dashboard-update
+  interval: 24h
```

```suggestion
  interval: 2h
```

Since this job has been failing for a bit, let's ratchet up the runs for a bit for a faster feedback loop.

---

done